CMakeLists.txt

```diff
@@ -273,6 +275,13 @@ macro(standardize_output_folder bin_target)
     PDB_OUTPUT_DIRECTORY "${CMAKE_BINARY_DIR}/bin")
 endmacro()
 
+if(OCOS_USE_CUDA)
+  include(ext_cuda)
```

Comment:
can we check if the system arch has GPU here and throw an error? so if a user sets OCOS_USE_CUDA to ON and tries building (accidentally) on a machine without a GPU they won't get failures they don't realize the origin for (similar to when a user tries to use ort gpu functionality without having onnxruntime-gpu installed).

Reply:
On building, it doesn't require CUDA, instead of CUDA runtime libraries.